apps/prairielearn/src/components/CourseRequestsTable.html.ts

```diff
@@ -209,7 +209,7 @@ function CourseRequestApproveForm({
   coursesRoot: string;
   csrfToken: string;
 }) {
-  const repo_name = 'pl-' + request.short_name.replace(' ', '').toLowerCase();
+  const repo_name = 'pl-' + request.short_name.replaceAll(' ', '').toLowerCase();
```

Comment:
`short_name` is currently restricted by a regexp to only contain a single space, but I figured it didn't hurt to change that if we ever want to relax that restriction.